src/libraries/Microsoft.PowerFx.Core/Public/CheckResult.cs

```diff
@@ -237,6 +251,8 @@ public HashSet<string> TopLevelIdentifiers
             }
         }
 
+        public ParserOptions ParserOptions => _parserOptions;

```

Comment:
>ParserOptions [](http://example.com/codeflow?start=29&length=13)

Why do we need to expose this? 
Is it encapsulated by the results of ParseResult?

Reply:
When we use TexlNode we don't have ParseResult